sklearn/linear_model/tests/test_base.py

```diff
@@ -754,17 +754,11 @@ def test_linear_regression_sample_weight_consistency(
     if fit_intercept:
         intercept_0 = reg.intercept_
     reg.fit(X[:-5], y[:-5], sample_weight=sample_weight[:-5])
-    if fit_intercept and sparse_container is None:
-        # FIXME: https://github.com/scikit-learn/scikit-learn/issues/26164
-        # This often fails, e.g. when calling
-        # SKLEARN_TESTS_GLOBAL_RANDOM_SEED="all" pytest \
-        # sklearn/linear_model/tests/test_base.py\
-        # ::test_linear_regression_sample_weight_consistency
-        pass
-    else:
-        assert_allclose(reg.coef_, coef_0, rtol=1e-5)
-        if fit_intercept:
-            assert_allclose(reg.intercept_, intercept_0)
+    # TODO: see if it fixes
+    # https://github.com/scikit-learn/scikit-learn/issues/26164
+    assert_allclose(reg.coef_, coef_0, rtol=1e-6)
```

Comment:
Are you sure you don't want to keep the previous `rtol`?

```suggestion
    assert_allclose(reg.coef_, coef_0, rtol=1e-5)
```

It's consistent with the relaxed rtol value a few lines below (at line 784).